apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -34,8 +39,17 @@ const App = (): JSX.Element => {
 		}
 	}, [isRoot, dispatch]);
 
+	const openModal = useCallback((): void => {
```

Comment:
In my opinion, using useCallback for functions like openModal and closeModal adds unnecessary complexity. The overhead of useCallback might not be worth it . For straightforward scenarios, defining functions directly can be simpler and more efficient.